### PR TITLE
Swope install upgrade

### PR DESCRIPTION
**Pull request by swopebe**
Need both topics for disconnected and main/intro file reviewed.

### CODE REVIEW

File: install/install_disconnected.adoc [hunk 1]
```diff
@@ -1,16 +1,15 @@
 [#install-on-disconnected-networks]
 = Install in disconnected network environments
 
-You might need to install {product-title} on {ocp} clusters that are not connected to the Internet. To install on a disconnected hub cluster, perform the following steps in addition to the usual install or upgrade steps for the connected network environment.
+You might need to install {product-title} on {ocp} clusters that are not connected to the Internet (disconnected). To install on a disconnected hub cluster, perform the following steps in addition to the usual install or upgrade steps that are for the connected network environment.
 
-*Required access:* You need cluster administration access for all installation tasks.
+*Required access:* You need _luster administration access for all installation and upgrade tasks.
```
Comment (oafischer) [inline install/install_disconnected.adoc]: Add missing c to cluster (and replace _ if unintentional).

File: install/install_disconnected.adoc [hunk 2]
```diff
@@ -41,33 +40,31 @@ Open the URL in your browser and check the result. If the console URL displays `
...
-https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/operators/index#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]in the {ocp-short} documentation.
+To prepare your disconnected cluster for installing {product-title}, follow the procedure that is described in
+https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/operators/index#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] in the {ocp-short} documentation.
 
 [#additional-requirements]
 === Additional requirements 
-
-When performing the previous procedures, note the the follow requirements specific to {product-title}:
+ 
+When you perform the previous procedures, note the the follow requirements that are also specific to {product-title}:
```
Comment (oafischer) [inline install/install_disconnected.adoc]: Remove duplicate "the"

Also check this sentence again, I think there is a word missing.

**Review by oafischer — APPROVED** [review]
Looks good! Just left a few comments for typos, thanks!

File: install/install_disconnected.adoc [hunk 3]
```diff
@@ -19,7 +18,7 @@ the xref:../install/requirements.adoc#requirements-and-recommendations[Requireme
 
 You must meet the following requirements before you install {product-title}:
 
-* Since you are installing in a disconnected network enviornment, you need access to a local image registry to hold mirrored OLM catalogs and operator images. You probably already set up a local image registry when installing the {ocp-short} cluster in this enviornment, so you should be able to use the same local image registry.
+* Since you are installing in a disconnected network environment, you need access to a local image registry to hold mirrored {olm} catalogs and operator images. You probably already set up a local image registry when installing the {ocp-short} cluster in this enviornment, so you should be able to use the same local image registry.
```
Comment (cadawson) [inline install/install_disconnected.adoc]: Would "store" be better than "hold"?

File: install/install_disconnected.adoc [hunk 4]
```diff
@@ -19,7 +18,7 @@ the xref:../install/requirements.adoc#requirements-and-recommendations[Requireme
...
 
 * You must have a workstation that has access to both the Internet, as well as your local mirror registry.
```
Comment (cadawson) [inline install/install_disconnected.adoc]: You must have a workstation that has access to both the Internet and your local mirror registry.

File: install/upgrade_disconnected.adoc [hunk 5]
```diff
@@ -1,77 +1,69 @@
...
-your OLM subscription for {product-title} to the upgrade channel for the new release.
-However, because of the special characteristics of the disconnected environment, address the following mirroring requirements
-before changing the update channel to start the upgrade process.
+As is the case for upgrading in a connected network environment, the upgrade process is started by changing the upgrade channel in
+your {olm} subscription for {product-title} to the upgrade channel for the new release. 
 
-. Ensure that required packages are updated in your mirror catalog
+However, because of the special characteristics of the disconnected environment, you need to address the following mirroring requirements before changing the update channel to start the upgrade process:
+
+. Ensure that required packages are updated in your mirror catalog.
```
Comment (cadawson) [inline install/upgrade_disconnected.adoc:18]: These are numbers. Would bullets be better?

File: install/install_disconnected.adoc [hunk 6]
```diff
@@ -19,9 +18,9 @@ the xref:../install/requirements.adoc#requirements-and-recommendations[Requireme
...
+* Since you are installing in a disconnected network environment, you need access to a local image registry to hold mirrored {olm} catalogs and operator images. You probably already set up a local image registry when installing the {ocp-short} cluster in this environment, so you should be able to use the same local image registry.
 
-* You must have a workstation that has access to both the Internet, as well as your local mirror registry. 
+* You must have a workstation that has access to both the Internet and your local mirror registry. 
 
 * A supported {ocp} version must be deployed in your environment, and you must be logged in with the command line interface (CLI). See the https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/installing/index[{ocp-short} version 4.11 install documentation] for information on installing {ocp}. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the CLI] for information about installing and configuring `oc` commands with the Red Hat OpenShift CLI.
```
Comment (cadawson) [inline install/install_disconnected.adoc:26]: In line 32, we say "When and if..." It seems a little squishy. Suggest using "When" if it is required to be connected. Suggest use "if", if you don't really have to be connected.

File: install/install_disconnected.adoc [hunk 7]
```diff
@@ -41,44 +40,43 @@ Open the URL in your browser and check the result. If the console URL displays `
...
 
 [#additional-requirements]
 === Additional requirements 
-
-When performing the previous procedures, note the the follow requirements specific to {product-title}:
+ 
+When you complete the previous procedures, note the following requirements that are also specific to {product-title}:
 
 [#disconnect-include-acm-pkgs]
 ==== Include required operator packages in your mirror catalog
```
Comment (cadawson) [inline install/install_disconnected.adoc]: The title is the same as the first sentence of the bullet. Is that intentional?

File: install/install_disconnected.adoc [hunk 8]
```diff
@@ -104,27 +102,27 @@ mirror:
...
+. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/operators/index#olm-restricted-networks-operatorhub_olm-restricted-networks[Disabling the default OperatorHub sources]
+. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/operators/index#olm-mirror-catalog_olm-restricted-networks[Mirroring the Operator catalog]
+. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/operators/index#olm-creating-catalog-from-index_olm-restricted-networks[Adding a catalog source for your mirrored catalog]
 
 [#note-source-name]
 ==== Find the catalog source name
 
-As described in the procedures in the {ocp} documentation, you need to add a `CatalogSource` resource to your disconnected cluster. Take note of the value of the `metadata.name` field, which you need later.
+As described in the procedures in the {ocp} documentation, you need to add a `CatalogSource` resource to your disconnected cluster. *Important:* Take note of the value of the `metadata.name` field, which you will need later.
```
Comment (cadawson) [inline install/install_disconnected.adoc:125]: This line looks redundant with line 141. See what you think.

**Review by cadawson — CHANGES_REQUESTED** [review]
Fixed some nits and added a few comments. Thanks!

File: install/install_disconnected.adoc (hunk 8 above)
Comment (swopebe) [reply install/install_disconnected.adoc:125]: Joe asked for a bit of redundancy based on where he is seeing people get stuck. This is one of those examples.

File: install/upgrade_disconnected.adoc (hunk 5 above)
Comment (swopebe) [reply install/upgrade_disconnected.adoc:18]: treated these at steps; they are somewhat in order.

File: install/install_disconnected.adoc (hunk 7 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: no--I just didn't catch it

File: install/install_disconnected.adoc (hunk 1 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: done

File: install/install_disconnected.adoc (hunk 2 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: done

File: install/install_disconnected.adoc (hunk 3 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: done

File: install/install_disconnected.adoc (hunk 4 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: done

File: install/install_disconnected.adoc (hunk 6 above)
Comment (swopebe) [reply install/install_disconnected.adoc:26]: done

File: install/install_disconnected.adoc (hunk 7 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: done

**Pull request merged**